markdown.dtx

```diff
@@ -25,11 +25,14 @@
 \def\theCodelineNo{\textcolor[gray]{0.5}{\oldCodelineNo}}
 \makeatletter\renewcommand\fps@figure{htbp}\makeatother
 % TODO replace with l3keys functions
-\setkeys{Gin}{
-  width = \columnwidth,
-  height = 0.65\paperheight,
-  keepaspectratio,
-}
+\ExplSyntaxOn
+\keys_set:nn { Gin }
+  {
+    width = \columnwidth,
+    height = 0.65\paperheight,
+    keepaspectratio,
+  }
+\ExplSyntaxOff
```

Comment:
`\setkeys{Gin}` configures the `graphicx` package. The graphicx package uses and will use `keyval`, i.e. this is not a useful test.

Reply:
Thanks, that explains a lot. :)